File: exercicios_C/1.exercSequenciais/exerc8.c
#include <stdio.h>

void main(){
    float val, horas = 0;
    printf("informe o valor da sua hora: ");
    scanf("%f", &val);
    printf("informe o numero de horas: ");
    scanf("%f", &horas);
    printf("O seu salario e: R$ %.2f", val * horas);
}

File: exercpythonbrasil/1.exerc_sequencial/exerc8.py
""" Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalhadas no mês.
Calcule e mostre o total do seu salário no referido mês.
"""
valor_hora = float(input("Informe o valor da hora trabalhada: "))
horas_trabalhadas = int(input("Informe o numero de horas trabalhadas: "))
print(f'O seu salário é: R${int(horas_trabalhadas * valor_hora)}.')
